docs/modules/storage/pages/configuring-persistence.adoc

```diff
@@ -1261,6 +1261,55 @@ config.addMapConfig(mapConfig);
 ====
 |===
 
+[[sql]]
+== SQL Metadata Options
+
+Use the following configuration to enable the cluster-wide persistence of SQL metadata to disk. With persistence enabled, SQL mappings, data connections, and views are retained after the cluster restarts.
+
+[tabs] 
+==== 
+XML:: 
++ 
+--
+[source,xml]
+----
+<hazelcast>
+  <map name="__sql.catalog">
```

Comment:
Same as the Java configuration, there is an option in the SQL Config specifically to enable SQL Catalog persistence in a safe way: 

```
<hazelcast>
    <sql>
        <catalog-persistence-enabled>true</catalog-persistence-enabled>
    </sql>
</hazelcast>
```

Reply:
Updated with correct config.